react.js

```diff
@@ -1,3 +1,15 @@
 module.exports = {
-  extends: ['./base.js', './naming/react', './overrides.js'],
+  extends: [
+    './base.js', 
+    './naming/react', 
+    './overrides.js',
+    'plugin:react/recommended',
+    'plugin:react/jsx-runtime',
+    'plugin:react-hooks/recommended'
+  ],
```

Comment:
Order matters here, so we should be putting our rules last otherwise we have no way of overriding the rules in the recommended rulesets.